lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/form/Auto.js

```diff
@@ -127,7 +127,7 @@ qx.Class.define("callbackery.ui.form.Auto", {
             if (s.set) {
                 if (s.widget == 'date') {
                     if (s.set.value) {
-                        var dateValue = s.set.value;
+                        var dateValue = s.set.value * 1000; // incoming epoch seconds
                         delete s.set.value;
```

Comment:
No. I just didn't have the * 1000 before. NOW it is consistent, I think. Search for 1000 in this file.